src/common-utils/install.sh

```diff
@@ -292,7 +292,7 @@ cat << 'EOF' > /usr/local/bin/systemctl
 #!/bin/sh
 set -e
 if [ -d "/run/systemd/system" ]; then
-    exec /bin/systemctl/systemctl "$@"
+    exec /bin/systemctl "$@"
```

Comment:
We'd need to bump the version for this Feature 😅

Reply:
Updated!